### DISCUSSION TITLE
runtimePublicPath 如何配置

### DISCUSSION THREAD

**Discussion by 763483000** [umijs/umi#8597] [Q&A]
![image](https://user-images.githubusercontent.com/6050691/179506909-e2cf30d5-10b6-4d28-a214-035a4edccb5a.png)
历史版本 这个是个boolean类型的值，当前变更了Object ，但是官网又没有示例，求助

**Comment by 763483000** [comment]
fatal - AssertionError [ERR_ASSERTION]: Invalid config values: runtimePublicPath
Invalid value for runtimePublicPath:
"value" must be of type object

**Comment by fz6m** [comment]
See https://github.com/umijs/umi/discussions/8421#discussioncomment-3171180
